scripts/update-version

```diff
@@ -50,4 +50,4 @@ echo "creating types for legacy code bases..."
 rm -f dist/rx-player.d.ts rx-player.min.d.ts
 echo "${copyright_text}
 import RxPlayer from \"./_esm5.processed/minimal\";
-export default RxPlayer;" | tee dist/rx-player.d.ts dist/rx-player.min.d.ts >/dev/null
+export default RxPlayer;" | tee dist/rx-player.d.ts dist/rx-player.min.d.ts > /dev/null
```

Comment:
nothing but github complaining about not having a new line at end of file

Reply:
I added a line because I don't mind much, so if it can make some website's robot happier, happy to help!